lib/css/CssParser.js

```diff
@@ -173,7 +174,9 @@ class CssParser extends Parser {
 		const module = state.module;
 		if (
 			module.type === CSS_MODULE_TYPE_AUTO &&
-			IS_MODULES.test(module.resourceResolveData.path)
+			IS_MODULES.test(
+				parseResource(module.matchResource || module.resource).path
```

Comment:
Why not `parser.state.module.resource`?

Reply:
Just added to the test about the `matchResource` part, only use `module.resource` won't correctly handle the matchResource feature